fab3/filters.go

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package fab3
+
+import (
+	"encoding/hex"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"github.com/hyperledger/fabric-chaincode-evm/fab3/types"
+)
+
+type filterEntry interface {
+	LastAccessTime() time.Time
+	Filter(s *ethService) ([]interface{}, error)
+}
+
+type logsFilter struct {
+	logArgs        *types.GetLogsArgs
```

Comment:
Because of how rebasing and I guess github works with comments, we lost the discussion on the GetLogsArgs naming. I am thinking something like `FilterArgs` or `LogFilterArgs` as the name would be good. But since this is a rather large PR already, either we should do it a different PR or at least in a separate commit.

Reply:
previous suggestion was logargs https://github.com/hyperledger/fabric-chaincode-evm/pull/11#discussion_r364001268

Reply:
Ohh, I see now. I miss gerrit, it would been able to highlight the right word. My comment was about the type name not the field name. I think the field name `logArgs` is fine. I do prefer that over `gla`